packages/editor/src/components/post-title/index.js

```diff
@@ -36,12 +36,21 @@ class PostTitle extends Component {
 		this.onUnselect = this.onUnselect.bind( this );
 		this.onKeyDown = this.onKeyDown.bind( this );
 		this.redirectHistory = this.redirectHistory.bind( this );
+		this.textareaRef = createRef();
 
 		this.state = {
 			isSelected: false,
 		};
 	}
 
+	componentDidMount() {
+		// If there is a post title and it's empty, we should focus the title so
+		// the user can start typing the title right away.
+		if ( ! this.props.title || ! this.props.title.length ) {
+			this.textareaRef.current.textarea.focus();
```

Comment:
Can `Textarea` receive an `autoFocus` prop to simplify the implementation?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#attr-autofocus

Reply:
I wonder if it supports passing through other props given it's not a stock `<Textarea>`, but I'll have a look.

Reply:
It can and that actually worked well, but Puppeteer doesn't seem to like it. I'm just working on figuring out why and then this'll be good to go and less complicated.